lnrpc/routerrpc/router_server.go

```diff
@@ -167,6 +170,7 @@ func (s *Server) Start() error {
 //
 // NOTE: This is part of the lnrpc.SubServer interface.
 func (s *Server) Stop() error {
+	close(s.quit)
```

Comment:
Does this need a waitgroup too, to wait for completion?

Reply:
Offline discussion about this, no reason for subscriptions not to be wait grouped (I thought there could be some grpc magic in the background). Added wait group for routerrpc subserver + `trackPayment`.

Reply:
do we want to add `sync.Once` here? o/w calling `Stop()` multiple times will panic